src/i18n/locales/en.json

```diff
@@ -879,6 +879,8 @@
         "preferLocalLyrics": "prefer local lyrics",
         "showLyricsInSidebar_description": "a panel will be added to the attached play queue that displays the lyrics",
         "showLyricsInSidebar": "show lyrics in player sidebar",
+        "showRatings_description": "controls if the ratings feature shows up in the interface",
+        "showRatings": "show arist/album ratings",
```

Comment:
Rename this string to `show star ratings`, because it's not only on artist/album, but also songs. (also there is a typo)